graphqlapi/dynamic_generation_meta_fetch.go

```diff
@@ -0,0 +1,650 @@
+
+	topOccurencesFields, err := genMetaClassDatePropertyTopOccurrencesFields(class, property)
+	if err != nil {
+		return nil, err
+	}
+	metaFetchDatePointingFields := graphql.Fields{
+		"type": &graphql.Field{
+			Name:        mergeStrings("Meta", class.Class, "MetaPointingTo"),
+			Description: "datatype of the property",
+			Type:        graphql.String,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+		"counter": &graphql.Field{
+			Name:        mergeStrings("Meta", class.Class, "MetaPointingFrom"),
+			Description: "total amount of found instances",
+			Type:        graphql.Int,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+		"topOccurences": &graphql.Field{
+			Name:        mergeStrings("Meta", class.Class, "MetaPointingFrom"),
+			Description: "most frequent property values",
+			Type:        topOccurencesFields,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+	}
+	metaFetchDateProperty := graphql.ObjectConfig{
+		Name:        mergeStrings("Meta", class.Class, property.Name, "Obj"),
+		Fields:      metaFetchDatePointingFields,
+		Description: "Property meta information",
+	}
+	return graphql.NewObject(metaFetchDateProperty), nil
+}
+
+func genMetaClassDatePropertyTopOccurrencesFields(class *models.SemanticSchemaClass, property *models.SemanticSchemaClassProperty) (*graphql.Object, error) {
+
+	metaFetchMetaPointingFields := graphql.Fields{
+		"value": &graphql.Field{
+			Name:        mergeStrings("Meta", class.Class, property.Name, "TopOccurencesValue"),
+			Description: "property value of the most frequent properties",
+			Type:        graphql.String,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+		"occurs": &graphql.Field{
+			Name:        mergeStrings("Meta", class.Class, property.Name, "TopOccurencesOccurs"),
+			Description: "number of occurrance",
+			Type:        graphql.String,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+	}
+	metaFetchMetaPointing := graphql.ObjectConfig{
+		Name:        mergeStrings("Meta", class.Class, property.Name, "TopOccurencesObj"),
+		Fields:      metaFetchMetaPointingFields,
+		Description: "most frequent property values",
+	}
+	return graphql.NewObject(metaFetchMetaPointing), nil
+}
+
+func genMetaClassCRefPropertyObj(class *models.SemanticSchemaClass, property *models.SemanticSchemaClassProperty) (*graphql.Object, error) {
+
+	metaCRefPointingObj, err := genMetaCRefPointingObj(class, property)
+	if err != nil {
+		return nil, fmt.Errorf("Could not build GraphQL schema, because: %v", err)
```

Comment:
Does this error message add something? Otherwise, just return the err from genMetaCrefPointingObj.

Reply:
Apparently genMetaCrefPointingObj does not ever return an error, so this can be removed.